src/main/java/com/easypost/easyvcr/requestelements/Response.java

```diff
@@ -52,256 +37,6 @@ public final class Response extends HttpElement {
      */
     private URI uri;
 
-    /**
-     * Build a CloseableHttpResponse out of this Response.
-     *
-     * @return a CloseableHttpResponse representation of this Response.
-     */
-    public CloseableHttpResponse toCloseableHttpResponse() {
```

Comment:
## Author's Note

This class (and a few other removed elements such as ProtocolVersion) were prematurely added for Apache HTTP client compatibility, but were never used. That plan has since been abandoned, and this dead code has been removed.